### PR TITLE
unit test for method CreateIssuedTokenOverTransportBindingElement

### PR DESCRIPTION
**Pull request by imcarolwang**
For #3760 .

### CODE REVIEW

File: src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs [hunk 1]
```diff
@@ -42,4 +42,26 @@ public static void Property_KeyEntropyMode()
...
     }
+
+    [WcfFact]
+    public static void Method_CreateIssuedTokenOverTransportBindingElement()
+    {
+        IssuedSecurityTokenParameters tokenParameters = new IssuedSecurityTokenParameters();
+        TransportSecurityBindingElement securityBindingElement = SecurityBindingElement.CreateIssuedTokenOverTransportBindingElement(tokenParameters);
+
+        Assert.Contains(tokenParameters, securityBindingElement.EndpointSupportingTokenParameters.Endorsing);
+        Assert.Equal(System.ServiceModel.MessageSecurityVersion.Default, securityBindingElement.MessageSecurityVersion);
```
Comment (StephenBonikowsky) [inline src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs]: Please do a one time manual verification that MessageSecurityVersion.Default is the same in Core as it is in Full Framework.

File: src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs [hunk 2]
```diff
@@ -42,4 +42,26 @@ public static void Property_KeyEntropyMode()
...
+        Assert.False(securityBindingElement.LocalClientSettings.DetectReplays);
+        Assert.True(securityBindingElement.IncludeTimestamp);
+
+        tokenParameters.KeyType = System.IdentityModel.Tokens.SecurityKeyType.BearerKey;
```
Comment (StephenBonikowsky) [inline src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs]: We don't yet want to light up SecurityKeyType.BearerKey
In the setter for KeyType could you throw `throw ExceptionHelper.PlatformNotSupported();`
https://github.com/dotnet/wcf/blob/master/src/System.Private.ServiceModel/src/System/ServiceModel/Security/Tokens/IssuedSecurityTokenParameters.cs#L101

You can add it to this same PR and then remove the checks you `if (issuedTokenParameters.KeyType == SecurityKeyType.BearerKey)`

File: src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs (hunk 1 above)
Comment (imcarolwang) [reply src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs]: Verified, it's the same as in Full Framework.

File: src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs (hunk 2 above)
Comment (imcarolwang) [reply src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs]: Addressed.

**Pull request merged**